deployment/deployZapIn.js

```diff
@@ -1,7 +1,9 @@
+const {runVerifyAndExport} = require('./helper');
+
 const ZapIn = artifacts.require('ZapIn');
 
-async function main () {
-  // ropsten 
+async function main() {
+  // ropsten
   const wethAddress = '0xc778417e063141139fce010982780140aa0cd5ab';
```

Comment:
should get the address from some configs or files, sometimes I forget to update the weth or factory address and deploy for other networks

Reply:
upgrade to use **hardhat-deploy** for better development